WcaOnRails/config/environments/development.rb

```diff
@@ -67,7 +67,7 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.after_initialize do
-    Bullet.enable = true
+    Bullet.enable = !ENVied.DISABLE_BULLET
```

Comment:
Is there a scenario in which we want to disable Bullet in development?

Reply:
I need to look into this. When I first started working on this page, I was running into messages from Bullet that I could not fully track down. I introduced this environment variable so I wouldn't have to waste time dismissing the bullet dialog. I was worried I might be running into something like https://github.com/jfly/worldcubeassociation.org/blob/6ad50b1c749d4140caca8f94d2781498dd7eea0a/WcaOnRails/config/environments/development.rb#L76-L78.